io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,54 @@
+id: io.neovim.nvim.tool.tmux
+build-extension: true
+sdk: org.freedesktop.Sdk//22.08
+runtime: io.neovim.nvim
+runtime-version: stable
+branch: stable
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: ${FLATPAK_DEST}
```

Comment:
this doesnt work, there is no shell variable substitution.

Reply:
Fixed https://github.com/flathub/flathub/pull/4119/commits/c55d968f1cd954dee52d4439b7c6da20a2108e2a